### ISSUE TITLE
Nikon COOLPIX P1000 white point (sometimes?) off

### ISSUE THREAD

**Issue by Thanatomanic** [Beep6581/RawTherapee#5734]
In #1755 @dimonoid provided camera samples with strong overexposure. All are shot at ISO100. Interestingly, in one case the sky is simply white (as expected), but in the other cases the sky is magenta. This indicates something is not correct with the white point setting of the camera in `camconst.json`. 

See #5480 for relevant recent commits for camera support.

**Comment by Thanatomanic** [comment]
I'm no expert on `camconst.json`, but this should do the trick I guess? 4093 -> 4087

```
    { // Quality B
        "make_model": "NIKON COOLPIX P1000",
        "dcraw_matrix": [ 14294, -6116, -1333, -1628, 10219, 1637, -14, 1158, 5022 ], // ColorMatrix2 from Adobe DNG Converter 11.4
        "ranges": { "black": 200, "white": 4087 }
    },

**Comment by heckflosse** [comment]
@Thanatomanic Sounds reasonable. Please commit your change :+1:

**Comment by Thanatomanic** [comment]
@dimonoid If you get a nightly build tomorrow, or build from source, at least part of your artifact problems will be gone!

**Comment by dimonoid** [comment]
I don't know if I should create a separate issue, so writing in here.
It looks like the black point is off too. There are some pixels with values 190, even a little bit lower. Because of that my dark frames didn't work as intended by introducing, not removing noise. 
Also white point looks really too high. I measured it 3903 at iso 100 to 3905 at iso 6400. I don't know where the previous number 4087 is coming from, it looks impossible.
Here is what I got:
```
{ // Quality B
        "make_model": "NIKON COOLPIX P1000",
        "dcraw_matrix": [ 14294, -6116, -1333, -1628, 10219, 1637, -14, 1158, 5022 ], // ColorMatrix2 from Adobe DNG Converter 11.4
        "ranges": { "black": 190, "white": 4096}//3903 instead of 4087, 190 instead of 200
    },
```
